src/js/themes/hpe.js

```diff
@@ -2110,7 +2110,7 @@ const buildTheme = (tokens, flags) => {
           }
           `,
       },
-      gap: components.hpe.radioButton.default.medium.gapX,
+      gap: {horizontal: 'xsmall'},
```

Comment:
```suggestion
      gap: components.hpe.radioButton.default.medium.gapX,
```
Keep this as it was since we have a component specific token